monai/apps/generation/maisi/networks/autoencoderkl_maisi.py

```diff
@@ -523,6 +515,8 @@ class MaisiEncoder(nn.Module):
         norm_eps: Epsilon for the normalization.
         attention_levels: Indicate which level from num_channels contain an attention block.
         with_nonlocal_attn: If True, use non-local attention block.
+        include_fc: whether to include the final linear layer in the attention block. Default to True.
```

Comment:
Please also update the docstring for num_res_blocks.
`num_res_blocks: Number of residual blocks (see ResBlock) per level.`
to
`num_res_blocks: Number of residual blocks (see AEKLResBlock) per level.`

Reply:
Done